source/controlTypes.py

```diff
@@ -662,21 +664,46 @@ class OutputReason(Enum):
 REASON_CARET = OutputReason.CARET
 REASON_ONLYCACHE = OutputReason.ONLYCACHE
 
-#: Text to use for 'current' values. These describe if an item is the current item 
-#: within a particular kind of selection.
-isCurrentLabels: Dict[Union[bool, str], str] = {
+
+class IS_CURRENT(Enum):
+	"""Values to use within NVDA to denote 'current' values.
+	These describe if an item is the current item within a particular kind of selection.
+	EG aria-current
+	"""
+	NO = "false"
+	Yes = "true"
+	PAGE = "page"
+	STEP = "step"
+	LOCATION = "location"
+	DATE = "date"
+	TIME = "time"
+
+	def getTranslationForUI(self):

```

Comment:
Alternative idea

```suggestion
	@property
	def translationForUI(self):
```

I think translation or translated would also be sufficient.